src/main/java/digital/slovensko/autogram/ui/gui/GUIApp.java

```diff
@@ -74,12 +79,14 @@ public void start(Stage windowStage) throws Exception {
 
         } catch (Exception e) {
             //ak nastane chyba, zobrazíme chybové okno a ukončíme aplikáciu
-            final var serverFinal = server; //pomocná premenná, do lambda výrazu nižšie musí vstupovať finalna premenná
+            var serverFinal = server; //pomocná premenná, do lambda výrazu nižšie musí vstupovať finalna premenná
```

Comment:
uz to nemusi byt final ako to pise v komente?

Reply:
musí "effectively final" alebo tak nejak. Čo je teda aj tento zápis. IntelliJ mi povedalo